common/src/test/java/org/apache/rocketmq/common/protocol/body/KVTableTest.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.rocketmq.common.protocol.body;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+public class KVTableTest {
+
+    @Test
+    public void testKVTable(){
```

Comment:
This test case needs to validate serialized and deserialized data

Reply:
Got it!